src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        int numberOfBalls = 3;
```

Comment:
Consider defining 'numberOfBalls' as a constant since it represents a configuration value that doesn't change at runtime. Constants in Java should be declared as 'static final' and their names should be in uppercase. For example: private static final int NUMBER_OF_BALLS = 3;